### PR TITLE
Add coverage reports to more applications

### PR DESCRIPTION
**Pull request by nickva**
Enable coverage reports for more applications

### CODE REVIEW

File: src/fabric/rebar.config [hunk 1]
```diff
@@ -1,15 +1,2 @@
-% Licensed under the Apache License, Version 2.0 (the "License"); you may not
```
Comment (eiri) [inline src/fabric/rebar.config]: I'm conflicted on removing licence header here, especially mixed with unrelated changes.

Comment (nickva) [reply src/fabric/rebar.config]: The other config files which configure coverage don't have license headers

Comment (eiri) [reply src/fabric/rebar.config]: Yes I understand it's inconsistent, but we want it to be at least a separate change so it'll be revertible without messing with unrelated changes, in case, say, it's legally should be other way around and it is all the other configs should have the licence header.
